src/webviews/webview-side/error-renderer/index.ts

```diff
@@ -43,9 +43,87 @@ const handleInnerClick = (event: MouseEvent, context: RendererContext<any>) => {
     }
 };
 
+function generateViewMoreElement(outputId: string) {
+    const container = document.createElement('span');
+    const preSpan = document.createElement('span');
+    preSpan.textContent = 'Output exceeds the ';
```

Comment:
This needs localization I think.  I believe we have the capability in the webview, as we pass the loc strings to it. At least we used to.

Reply:
And the loc makes it more complicated as you need the capability for the loc people to reorder the text.

Reply:
@rchiodo any code pointers for loc code used in the project?

Reply:
I think this should still work?
https://github.com/microsoft/vscode-jupyter/blob/1455d0b25b205c4d4672dfa7254d0e2bc7aa7be6/src/webviews/webview-side/react-common/locReactSide.ts#L10

That function is callable in our webviews, but maybe it's not wired up in the renderer yet. So you might have to send the same message across to the renderer where it can save the loaded collection.